src/App.vue

```diff
@@ -687,7 +689,25 @@ export default class App extends Mixins(
 
   /** Stores bootstrap item in the Pending List. */
   storeBootstrapPending (response: any): void {
-    this.storeBootstrapFiling(response) // *** TODO: implement this
+    const filing = response.filing as TaskTodoIF
+    // NB: these were already validated in storeBootstrapItem()
+    const header = filing.header
+    const data = filing[header.name]
+
+    // set addresses
+    this.storeAddresses({ data: data.offices || [] })
+
+    // set parties
+    this.storeParties({ data: { parties: data.parties || [] } })
```

Comment:
Although this bootstrap filing isn't complete, we  want to show the addresses and parties (both disabled) on the page, ie:

![image](https://github.com/user-attachments/assets/ddfe5402-1799-4167-9fab-02e1831f5b41)